feat: Add Ember RealWorld section (#390)
ember-learn/ember-blog: source/2019-11-29-the-ember-times-issue-126.md

Represent Ember in the State of JavaScript 2019 survey 📃,
Give thanks to Octane Guides contributors 🙏,
Ember + Tailwind CSS tips and tricks 🎨,

> Ember just got more real with RealWorld 🌟,

Reviewer: The Alex lint is failing because of the word just. You may want to try adding an exception, maybe before line 10 (Represent Ember), or skip the word if it's too much trouble?

Reviewer 2: Yeah, I think it would be great to skip the word then to make the build pass :+1:

> Ember just got more real with RealWorld 🌟,

Reviewer 2: ```suggestion
Ember got more real with RealWorld 🌟,
```

---

## [Ember RealWorld App is Feature Complete 🌠](https://github.com/gothinkster/ember-realworld)

[RealWorld](https://github.com/gothinkster/realworld), "The mother of all demo apps", is an Open Source initiative to showcase a full stack app build using any frontend or backend framework/library. It provides a spec for both the API and the frontend. The idea is to show how a "real world" app can be built using different frameworks/libraries.

> The Ember Realworld implementation is finally complete! It was started over two years ago by [@Alonski](https://github.com/Alonski) and after all this time is finally feature complete. A large number of contributors helped out along the way: [@jonnii](https://github.com/jonnii), [@alexlafroscia](https://github.com/alexlafroscia), [@delusioninabox](https://github.com/delusioninabox), [@patocallaghan](https://github.com/patocallaghan), [@GCheung55](https://github.com/GCheung55), [mansona](https://github.com/mansona), [@johpol](https://github.com/johpol)

Reviewer: If you'd like, I changed Realworld to RealWorld, and added @ to mansona.
```suggestion
The Ember RealWorld implementation is finally complete! It was started over two years ago by [@Alonski](https://github.com/Alonski) and after all this time is finally feature complete. A large number of contributors helped out along the way: [@jonnii](https://github.com/jonnii), [@alexlafroscia](https://github.com/alexlafroscia), [@delusioninabox](https://github.com/delusioninabox), [@patocallaghan](https://github.com/patocallaghan), [@GCheung55](https://github.com/GCheung55), [@mansona](https://github.com/mansona), [@johpol](https://github.com/johpol)
```

> The implementation showcases a class Ember.js app using the most idiomatic Ember patterns to serve as an example for the wider community. Feel free to look through the codebase and even offer suggestions, PRs or comments regarding how the app is built. There is always room for improvements. We are definitely looking for performance improvements so if thats your cup of tea ☕ please contribute.

Reviewer 2: What do you think about the following? I think it'd be

```suggestion
The implementation showcases a classic Ember.js app using the most idiomatic Ember patterns to serve as an example for the wider community. Feel free to look through the codebase and even offer suggestions, PRs or comments regarding how the app is built. There is always room for improvements. We are, for example, looking for performance improvements, so if that's your cup of tea ☕, please contribute.
```

> The RealWorld project has a rating system based on GitHub stars. Currently Ember RealWorld has 58 stars while the React/Redux implementation has over 3000. Can we band together and show that Ember is actually used in the "RealWorld"? Go and vote with your stars 🌟!

Reviewer 2: I really like that you convey the idea that the community is already showing its support for the project with almost 60 stars, but I personally think it'd be nice to focus on what this specific project could be without the direct comparison to React - which I believe would make it sound less competitive and even more agreeable.

What do you think about something along the lines of:

```suggestion
The RealWorld project has a rating system based on GitHub stars. Currently Ember RealWorld has been starred by community members 58 times, but we believe there could be so many more! To make the repo reflect the real world usage of Ember on the web, go and vote with your stars 🌟!
```

Let me know what you think!

> The RealWorld project has a rating system based on GitHub stars. Currently Ember RealWorld has been starred by community members 58 times, but we believe there could be so many more! To make the repo reflect the real world usage of Ember on the web, go and vote with your stars 🌟!

Reviewer: Can we link to the Ember repo again for convenience?
```suggestion
The RealWorld project has a rating system based on GitHub stars. Currently Ember RealWorld has been starred by community members 58 times, but we believe there could be so many more! To make the repo reflect the real world usage of Ember on the web, [go and vote with your stars](https://github.com/gothinkster/ember-realworld) 🌟!
```
